packages/front-end/components/Experiment/EditDOMMutationsModal.tsx

```diff
@@ -13,19 +13,23 @@ const EditDOMMutatonsModal: FC<{
   const [newVisualChange, setNewVisualChange] = useState<VisualChange>(
     visualChange
   );
+  const [css, setCss] = useState(visualChange.css);
+  const [js, setJs] = useState(visualChange.js);
```

Comment:
Why have separate state for these?  Everywhere you're updating state, you are also having to call `setNewVisualChange` anyway

Reply:
done